AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1773,6 +1773,8 @@ revouninstaller.com,logincasino.com,borsagundem.com,officelife.media,kod.ru,mast
 ! ###mc_embed_signup
 kosicednes.sk,tinyprojects.dev,cambio16.com,voxukraine.org,pnas.org,cachevalleydaily.com,bisnis.com,ilpost.it,bsblog.info,alcoparty.com.ua,codastory.com,informazionefiscale.it,kerch.fm,cinemablend.com,prostofinovo.ru,comss.ru,majorgeeks.com,buhgalter.com.ua,providr.com,24tv.ua,babylonbee.com,dexerto.com,football365.com,menrec.com,netdoktor.de,ottawamatters.com,retail-insider.com,walterfootball.com,wweek.com,zozhnik.ru###mc_embed_signup
 !
+theblaze.com##.newsletter-2020
+theblaze.com##div[id*="Newsletter_Post_"]
```

Comment:
What about this for all cases?
`theblaze.com##div[id*="_Newsletter_"]`

Reply:
It may be too wide and cause incorrect blocking.

Reply:
I checked a lot of pages so I think we can change these 2 rules to 1

Reply:
Repaired.